empress/clumpr/Statistics.py

```diff
@@ -0,0 +1,98 @@
+# Statistics.py
+# Computes statistics and creates cost histogram
+# Ran Libeskind-Hadas, June 2020
+
+# Performs Monte Carlo simulation to compute empirical p-value and histogram
+# of solution costs.
+# https://www.ncbi.nlm.nih.gov/pmc/articles/PMC379178/
+
+from clumpr import DTLReconGraph
+import random
+import matplotlib
+matplotlib.use('tkagg') # need this so plt.show() works
+import matplotlib.pyplot as plt
+
+from typing import List, Dict, Tuple
+
+def trials(host_tree: dict, parasite_tree: dict, phi: dict, dup_cost: float, transfer_cost: float, loss_cost: float, num_trials: int) -> list:
+    """
+    :param host_tree: dictionary representation of host tree
+    :param parasite_tree: dictionary of parasite tree
+    :param phi: dictionary representation of parasite tip to host tip mapping
+    :param dup_cost: float duplication cost
+    :param transfer_cost: float transfer cost
+    :param loss_cost: float loss cost
+    :param num_trials: int number of trials in Monte Carlo simulation
+    :return: list of floating point costs of reconciliations of the Monte Carlo samples
+    """
+   
+    costs = list()  # List of costs of random trials
+    parasites = list(phi.keys())
+    hosts = list(phi.values())
```

Comment:
same here

Reply:
This once is necessary because we are using random.choice(hosts).